inst/@sym/sym.m

```diff
@@ -277,7 +277,6 @@
   end
 
   asm = {};
-  check = true;
```

Comment:
Ok, I think I see, thank you.

There are paths in #590 that end early with `s = meh...; return`, so I don't see this PR's code as being much worse.  I will merge for now, fight with classdef when we get there ;-)